packages/platform-browser/src/dom/events/hammer_gestures.ts

```diff
@@ -135,7 +135,7 @@ export class HammerGestureConfig {
   };
 
   /**
-   * Creates a [HammerJS Manager](http://hammerjs.github.io/api/#hammer.manager)
+   * Creates a [HammerJS Manager](https://hammerjs.github.io/api/#hammer-manager)
```

Comment:
This doesn't work for me. `#hammermanager` (without the hyphen) does:

```suggestion
 * An injectable [HammerJS Manager](https://hammerjs.github.io/api/#hammermanager)
```

Reply:
Looks like there is a copy-paste error (you also changed the start of the line); I will revert that in a subsequent commit.

Reply:
Ooops 😇